front/src/components/UI/AuctionView/AuctionBlock.tsx

```diff
@@ -4,17 +4,19 @@ import { AuctionViewTitle } from "./AuctionView.styled";
 
 export interface AuctionProps {
   key: string;
-  siteId: string;
-  title: string;
-  imgSrc: string;
+  name:string
+  products: [id:string, name:string]
```

Comment:
tutaj to aż musiałem sprawdzić co się stało. Dlaczego products to zawsze dwuelementowa tablica z nazwami obu elementów?